lib/presentation/extensions/send_file_extension.dart

```diff
@@ -65,7 +67,30 @@ extension SendFileExtension on Room {
     final tempEncryptedFile =
         await File('${tempDir.path}/$formattedDateTime${fileInfo.fileName}')
             .create();
+    final tempThumbnailFile = await File(
+      '${tempDir.path}/$formattedDateTime${fileInfo.fileName}_thumbnail.jpg',
+    ).create();
+    final tempEncryptedThumbnailFile = await File(
+      '${tempDir.path}/$formattedDateTime${fileInfo.fileName}_encrypted_thumbnail',
```

Comment:
you shouldn't put encrypted_thumbnail last, it lose your mimeType

Reply:
what is your suggestion?
encrypted file still need mimeType?
we keep mimeType in the thumbnailFileInfo, and keep it in the event content json, so the mimeType won't lose